### ISSUE TITLE
训练停在启动阶段不动了

### ISSUE THREAD

**Issue by zenoir** [None9527/None_Z-image-Turbo_trainer#1]
ai-toolkit训练的效果不太满意，非常想尝试这个训练器。可是训练总是停留在启动状态，无法正式开始训练怎么办？我的显卡是4070tis。所有的依赖都安装了，包括torch、flash attention、diffusers。正常生成缓存。训练启动后也没有任何报错。主要是显存几乎跑满，占用15.5g左右，显卡占用率99%。但是显卡核心温度只有不到50度，根据我的经验这表明显卡根本没有跑起来，正常推理或训练时核心温度都在60度以上。应该是显存不足的缘故。之前训练wan都是使用musubi-tuner开启blocks_to_swap=24，显存占用基本在12G不到，训练就很正常。这个训练器我看到有block_swap，但是文件注释“16GB LoRA 训练不需要，开启反而可能卡死”，所以不知道该不该开。gradient_accumulation_steps已经设定为4了。

**Comment by zenoir** [comment]
这是我的配置文件。

> {
  "name": "mearizit",
  "model_type": "zimage",
  "acrf": {
    "enable_turbo": true,
    "turbo_steps": 10,
    "shift": 3,
    "jitter_scale": 0.02,
    "use_dynamic_shifting": true,
    "base_shift": 0.5,
    "max_shift": 1.15,
    "snr_gamma": 5,
    "snr_floor": 0.1,
    "use_anchor": true,
    "raft_mode": false,
    "free_stream_ratio": 0.3,
    "l2_schedule_mode": "constant",
    "l2_initial_ratio": 0.3,
    "l2_final_ratio": 0.3,
    "l2_milestones": "",
    "l2_include_anchor": false,
    "l2_anchor_ratio": 0.3,
    "latent_jitter_scale": 0
  },
  "network": {
    "dim": 32,
    "alpha": 16
  },
  "lora": {
    "resume_training": false,
    "resume_lora_path": "",
    "train_adaln": false,
    "train_norm": false,
    "train_single_stream": false
  },
  "optimizer": {
    "type": "AdamW8bit",
    "learning_rate": "1e-4"
  },
  "training": {
    "output_name": "mearizitone_v1",
    "learning_rate": 0.0001,
    "learning_rate_str": "1e-4",
    "weight_decay": 0.01,
    "lr_scheduler": "cosine",
    "lr_warmup_steps": 20,
    "lr_num_cycles": 1,
    "lambda_l1": 1,
    "lambda_cosine": 0,
    "enable_freq": true,
    "lambda_freq": 0.3,
    "alpha_hf": 1,
    "beta_lf": 0.2,
    "enable_style": false,
    "lambda_style": 0.3,
    "lambda_struct": 1,
    "lambda_light": 0.5,
    "lambda_color": 0.3,
    "lambda_tex": 0.5,
    "lambda_fft": 0,
    "loss_mode": "standard"
  },
  "dataset": {
    "batch_size": 1,
    "shuffle": true,
    "enable_bucket": true,
    "datasets": [
      {
        "cache_directory": "D:\\None_Z-image-Turbo_trainer\\datasets\\mearizit_v1",
        "num_repeats": 2,
        "resolution_limit": 1024
      }
    ]
  },
  "reg_dataset": {
    "enabled": false,
    "weight": 1,
    "ratio": 0.5,
    "datasets": []
  },
  "advanced": {
    "max_grad_norm": 1,
    "gradient_checkpointing": true,
    "blocks_to_swap": 16,
    "num_train_epochs": 20,
    "save_every_n_epochs": 2,
    "gradient_accumulation_steps": 1,
    "mixed_precision": "bf16",
    "seed": 42,
    "num_gpus": 1,
    "gpu_ids": "",
    "block_swap": "12"
  },
  "modified": "2025-12-12T15:42:46.001894",
  "created": "2025-12-08T09:21:01.027114"
}

**Comment by zenoir** [comment]
我用musubi-tuner试了，fp8模式下可以炼，效果还不错，和非量化模式下区别不大。
